sphinx/builders/epub3.py

```diff
@@ -157,30 +169,23 @@ def build_navlist(self, nodes):
                 continue
             if node['level'] > self.config.epub_tocdepth:
                 continue
-            if node['level'] == level:
-                navlist.append(self.new_navlist(node, level))
-            elif node['level'] == level + 1:
-                navstack.append(navlist)
-                navlist = []
-                level += 1
-                if lastnode and self.config.epub_tocdup:
```

Comment:
I agree.

2016年8月7日(日) 20:34 Takeshi KOMIYA <notifications@github.com>:

> In sphinx/builders/epub3.py
> <https://github.com/sphinx-doc/sphinx/pull/2806#discussion_r73800360>:
>
> > @@ -157,30 +169,23 @@ def build_navlist(self, nodes):
> >                  continue
> >              if node['level'] > self.config.epub_tocdepth:
> >                  continue
> > -            if node['level'] == level:
> > -                navlist.append(self.new_navlist(node, level))
> > -            elif node['level'] == level + 1:
> > -                navstack.append(navlist)
> > -                navlist = []
> > -                level += 1
> > -                if lastnode and self.config.epub_tocdup:
>
> @shimizukawa <https://github.com/shimizukawa> How about replacing the
> default EPUB builder by epub3 on Sphinx-1.5? If agreed, the removal means
> the epub_tocdup option will be deprecated in next major release. Are you
> okay?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sphinx-doc/sphinx/pull/2806/files/7977fc8e62edcf097123e82e9e8b5c2d8d1c7fec#r73800360>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJQR_Ndu_aMxXUO30sQKoOIM76xSMcqks5qdcK6gaJpZM4JRZqf>
> .
>